ydb/core/tx/schemeshard/schemeshard_build_index_tx_base.cpp

```diff
@@ -274,6 +274,9 @@ void TSchemeShard::TIndexBuilder::TTxBase::Fill(NKikimrIndexBuilder::TIndexBuild
         case NKikimrSchemeOp::EIndexType::EIndexTypeGlobalAsync:
             *index.mutable_global_async_index() = Ydb::Table::GlobalAsyncIndex();
             break;
+        case NKikimrSchemeOp::EIndexType::EIndexTypeGlobalVectorKmeansTree:
+            //TODO build vector index
```

Comment:
We can set settings here, same as for other indexes

Reply:
I intentionally placed TODO here in order to let you write code for building index. Nevertheless, I have removed TODO.